tests/lib/cli-engine.js

```diff
@@ -842,6 +842,15 @@ describe("CLIEngine", function() {
             assert.isFunction(formatter);
         });
 
+        if (process.platform === "win32") {
```

Comment:
We don't do tests like this because it's never run on Travis so we will never pick up a regression.